---
title: CV
---

<!DOCTYPE html>
<html>

<head>
  <meta charset="utf-8">
  <title>{{ page.title }}</title>
</head>

<body>
  <!-- Download Link -->
  <!-- <div class="bodyText" style='margin-top: 30px; margin-bottom: 20px;'>
    <a href="/assets/pdfs/cv.pdf"><h2><u>Download CV as PDF</u></h2></a>
  </div> -->

  <!-- CV in HTML -->
  <div class="container bodyText">

    <!-- Summary -->
    <h1 class="subheading">Summary</h1>
    <p>
      I am a scientist turned software engineer - my mission is to empower scientists by crafting high-quality software and building maintainable, scalable infrastructure. I currently work as a Bioinformatics Software Engineer at LanzaTech where I develop bioinformatics software and pipelines that empower scientists to engineer microorganisms that can produce useful chemicals sustainably.
    </p>
    <p>
      I currently live in the Chicago area with my wife, <u><a href="https://www.ayanastrutz.com/">Ayana</a></u>, and our two cats, Miko and Lupin.
    </p>

    See below and check out my <a href={{ site.data.links.linkedin }}><u>LinkedIn</u></a> for more info.

    <!-- Skills -->
    <h1 class="subheading">Skills: What can I do?</h1>
    I've loosely divided my skillset into four categories:
    <li style="margin-bottom: 0"><b>Software Development</b> - Crafting high-quality code to support scientific research</li>
    <li style="margin-bottom: 0"><b>Data Infrastructure</b> - Building and maintaining scalable data infrastructure</li>
    <li style="margin-bottom: 0"><b>Scientific Background</b> - Using my scientific background to inform the development process</li>
    <li><b>Other</b> - Miscellaneous skills and interests</li>

    <h3>Software Development</h3>
    <ul>
      <li><b>🧬 Scientific Software Development:</b> In my current role, I develop and maintain a suite of 10+ bioinformatics pipelines comprising 100+ (mostly open-source) bioinformatics tools to process genomics datasets. I also built a web app that constructs 1000s of mathematical models daily and validates these models against in-house scientific datasets.</li>
      <li><b>👨🏻‍💻 Write Web App Backends & APIs: </b>I've written and contributed to many web backends & REST APIs - I've used flask, Django, Next.js, and FastAPI (which I love using for quick prototypes).</li>
      <li><b>💻 Contribute to Frontend Development:</b> I'm familiar with vanilla JavaScript, Typescript, as well as basic HTML and CSS. I've built simple user interfaces for APIs I've developed. I built this site using Jekyll + HTML/CSS (<u><a href="https://github.com/jonstrutz11/jonstrutz11.github.io">code here</a></u>). I have also contributed to projects that use Vue on the frontend.</li>
      <li><b>🧠 Deploy AI & LLM Tools:</b> I've built RAG pipelines (using <u><a href=https://github.com/deepset-ai/haystack>Haystack</a></u>) to allow users to quickly query 1000+ internal documents supporting critical business processes. I've also made some minor <u><a href="https://github.com/deepset-ai/haystack/pull/8232">contributions</a></u> to Haystack itself.</li>
      <li><b>🐳 Containerize Apps:</b> I frequently use Docker (often with Docker Compose) to containerize applications to improve reproducibility.</li>
      <li><b>🐚 Write Shell Scripts:</b> I am very comfortable with bash scripting (which is particularly common in bioinformatics). I have also dabbled in bat scripting for Windows.</li>
      <li><b>🧹 Collaborate Using git & GitHub:</b> I use git & GitHub daily to submit issues, make commits, merge changes, and review and submit pull requests. I like to use interactive rebase to keep the commit history clean. <u><a href=https://github.com/jesseduffield/lazygit>Lazygit</a></u> is my best friend.</li>
      <li><b>✍🏻 Communicate and Document Effectively:</b> I prioritize writing readable code in order to clearly communicate the code's purpose. The joy I get from cleaning up messy code is like no other and highly motivating. I also use tools like Sphinx to write API documentation for python libraries.</li>
    </ul>
    <h3>Data Infrastructure</h3>
    <ul>
      <li><b>🛠️ Build Maintainable Data Infrastructure: </b>In my current role, I used Ansible to rearchitect 3 on-prem linux servers used for compute-heavy bioinformatics workloads - I designed a standardized data architecture, optimized storage resources, improved security, and increased reproducibility of bioinformatics pipelines.</li>
      <li><b>💾 Design and Query Databases:</b> I have experience designing small databases (~5-10 tables) from scratch. I have also worked with large legacy databases (100+ tables). I have experience with PostgreSQL, MySQL, as well as MongoDB.</li>
      <li><b>☁ Optimize AWS Resources:</b> I use AWS daily (especially EC2, ECS, S3, and RDS), and I am always looking for ways to optimize existing infrastructure (In my current role, I reduced costs for some AWS accounts by more than 50% while meeting business requirements). In 2025, I passed the AWS Solutions Architect exam at the Associate level.</li>
      <li><b>🔒 Prioritize Cybersecurity:</b> I've configured on-prem linux servers with security measures like SSH hardening and proper permissions & access control. I've implemented SSO on web app frontends and backends. See <u><a href=https://github.com/huggingface/chat-ui/pull/1631>this PR</a></u> where I added a new auth-related feature to an open-source LLM chat app.</li>
      <li><b>📈 Build Dashboards to Bolster Observability: </b>I've used the OpenTelemetry Collector to send metrics to Grafana cloud. I've built dashboards in Grafana to track key metrics for products I (and others) have built. I've also built Grafana dashboards to monitor resource usage of on-prem servers.</li>
      </ul>
    <h3>Scientific Background</h3>
    <ul>
      <li><b>🫱🏻‍🫲🏽 Communicate Effectively (especially with Scientists):</b> With a STEM PhD and as a former scientist myself, I can collaborate effectively with key stakeholders on science teams during the development process.</li>
      <li><b>✍🏻 Scientific Writing:</b> I clearly communicate my science to the rest of the world or company. <u><a href="https://scholar.google.com/citations?user=QSoCVUUAAAAJ&hl=en&authuser=1">My work</a></u> has been published in <i>Bioinformatics</i>, <i>PLoS Computational Biology</i>, and <i>Nature Biotechnology</i>.</li>
      <li><b>📊 Data Visualization:</b> I love thinking about ways to visualize data to meet user requirements. I am experienced with python libraries such as matplotlib, seaborn, and bokeh. I have also used tools like streamlit to create interactive dashboards.</li>
      <li><b>🏭 High-Performance Computing:</b> I have used the Slurm Workload Manager to submit jobs on supercomputing clusters. For example, I used the Quest supercomputing cluster at Northwestern University to predict MS2 spectra for millions of molecules in parallel.</li>
      <li><b>💭 Design & Analysis of Experiments:</b> I enjoy working with scientists to design experiments and analyses that best answer the most pressing questions. I have experience with both frequentist statistics as well as Bayesian statistical modeling.</li>
      <li><b>🧮 Mathematical Modeling:</b> I have used linear programming techniques to make metabolic state predictions for bacteria. I have also developed kinetic metabolic models, using both deterministic (e.g. ODE) and stochastic (e.g. Kinetic Monte Carlo) methods.</li>
      <li><b>🤖 Machine Learning:</b> During my PhD, I built a random forest regressor model using scikit-learn to predict chromatographic retention times in order to help identify unknown molecules in a metabolomics dataset.</li>
      <li><b>🧬 Bioinformatics:</b> I have experience processing and analyzing large genomics & transcriptomics datasets (e.g. alignment, variant calling, assembly & annotation). I also have experience processing and annotating metabolomics datasets.</li>
      <li><b>🧪 Cheminformatics:</b> I have used tools like RDKit and mordred to process, analyze, and interpret chemical data.</li>
    </ul>
    <h3>Other</h3>
    <ul>
      <li><b>📱 Develop iOS applications:</b> I develop iPhone apps on the side - currently, the apps I've built just live on my phone (I build them primarily for my own use), but I might release some in the future (see blog for more details).</li>
      <li><b>🏋️‍♂️ Strength Training:</b> I love optimizing my strength training protocols. I've been lifting weights for 10+ years and am obsessed with the science behind strength gain and muscle growth.</li>
      <li><b>🇯🇵 Japanese:</b> I've been learning Japanese for 10+ years now (and I still suck, but I enjoy it). I've put in over 1,000 hours reading & listening to Japanese.</li>
    </ul>

    <!-- Professional Experience -->
    <h1 class="subheading">Professional Experience</h1>
    <b>LanzaTech (Skokie, IL)</b> - Feb 2024 - Now
    <li>Bioinformatics Software Engineer, AI & Computational Biology</li>

    <b>LanzaTech (Skokie, IL)</b> - July 2022 - Jan 2024
    <li>Scientist, AI & Modeling</li>

    <b>Northwestern University (Evanston, IL)</b> - 2016 - 2022
    <li>PhD Candidate in Chemical & Biological Engineering</li>

    <b>LanzaTech (Skokie, IL)</b> - June 2018 - Dec 2018
    <li>Computational Biology Intern</li>

    <b>Wood Laboratory at Ohio State (Columbus, OH)</b> - 2014 - 2016
    <li>Undergraduate Researcher</li>

    <b>Ohio State University (Columbus, OH)</b> - 2013 - 2015
    <li>Undergraduate Teaching Assistant</li>

    <b>DuPont (Circleville, OH)</b> - Summer 2013
    <li>Manufacturing Technology Engineer Intern</li>

    <!-- Education -->
    <h1 class="subheading">Education</h1>
    <b>Northwestern University (Evanston, IL)</b> - Sep 2016 - June 2022
    <li style="margin-bottom: 0">Ph.D. Chemical & Biological Engineering</li>
    <li style="margin-bottom: 0">Co-advised by Keith Tyo & Linda Broadbelt</li>
    <li><i>Computational Approaches for Enhanced Metabolomics Annotation and DNA-based Biorecording and Data Storage</i></li>

    <b>Ohio State University (Columbus, OH)</b> - Sep 2011 - Dec 2015
    <li style="margin-bottom: 0">B.S. Chemical & Biomolecular Engineering</li>
    <li style="margin-bottom: 0">Biomedical Engineering Minor</li>
    <li>German Minor</li>

    <b>Circleville High School (Circleville, OH)</b> - 2007 - 2011
    <li>Valedictorian</li>

    <!-- Honors & Awards -->
    <h1 class="subheading">Honors & Awards</h1>
    <li style="margin-bottom: 0">ARCS Scholar Award (renewed twice) - 2017, 2018, 2019</li>
    <li style="margin-bottom: 0"><b>Biotechnology Training Program Trainee</b> - 2017, 2018</li>
    <li style="margin-bottom: 0"><b>Undergraduate Research Award</b> - 2015</li>
    <li style="margin-bottom: 0">Honda-OSU Math Medal Award - 2011</li>
    <li style="margin-bottom: 0">Columbus Dispatch Scholar-Athlete Runner-Up Male</b> - 2011</li>
    <li style="margin-bottom: 0">National Merit Finalist - 2011</li>
    <li>High School Valedictorian - 2011</li>

    <!-- Teaching Experience -->
    <h1 class="subheading">Teaching Experience</h1>
    <b>Northwestern University (Evanston, IL)</b>
    <li style="margin-bottom: 0">ChBE 373 - Global Health & Biotechnology (TA) - F2019</li>
    <li>ChBE 477 - Bioseparations (TA) - S2017, S2018, S2019</li>
    
    <b>Ohio State University (Columbus, OH)</b>
    <li>CBE 2420 - Transport Phenomena I - F2013, F2014, F2015</li>

    <!-- Organizations -->
    <h1 class="subheading">Organizations</h1>
    <b>Northwestern University (Evanston, IL)</b>
    <li>Graduate Student Forum - 2017 - 2018</li>

    <b>Ohio State University (Columbus, OH)</b>
    <li>Phi Kappa Tau Fraternity, Gamma Chapter
      <ul style="margin-top: 0;">
        <li style="margin-bottom: 0">Vice President - 2013 - 2015</li>
        <li style="margin-bottom: 0">Secretary - 2012</li>
        <li style="margin-bottom: 0">Founding Father - 2012</li>
      </ul>
    </li>

    <!-- Publications -->
    <h1 class="subheading">Publications</h1>
    
    <div style="font-size: 16px;">
      <b>Primary Research</b>
      <li>Callisto A, <b>Strutz J</b>, Leeper K, Kalhor R, Church G, Tyo KEJ, Bhan N (2024)
      Post-translational digital data encoding into the genomes of mammalian cell populations.
      <i>bioRxiv</i> (in review)</li>

      <li>Shebek K, <b>Strutz J</b>, Broadbelt L, Tyo K (2023)
      Pickaxe: a Python library for the prediction of novel metabolic reactions.
      <i>BMC Bioinformatics</i> 24:106</li>

      <li><b>Strutz J</b>, Shebek K, Broadbelt L, Tyo K (2022)
      MINE 2.0: enhanced biochemical coverage for peak identification in untargeted metabolomics.
      <i>Bioinformatics</i>, 38(13):3484-3487.</li>

      <li>Liew EF, Nogle R, Abdalla T, Rasor BJ, Canter C, Jensen RO, Wang L, <b>Strutz J</b>, ..., Köpke M (2022)
      Carbon-negative production of acetone and isopropanol by gas fermentation at industrial pilot scale.
      <i>Nature Biotechnology</i>, 40:335-344.</li>

      <li>Bhan NJ, Callisto A, <b>Strutz J</b>, Glaser J, Kalhor R, Boyden E, Church G, Kording K, Tyo KEJ (2021)
      Recording Temporal Signals with Minutes Resolution Using Enzymatic DNA Synthesis.
      <i>Journal of the American Chemical Society</i>, 143:16630-16640.</li>
      
      <li>St. John P, <b>Strutz J</b>, Broadbelt LJ, Tyo KEJ, Bomble YJ (2019)
      Bayesian inference of metabolic kinetics from genome-scale multiomics data.
      <i>PLoS Computational Biology</i>, 15(11):e1007424.</li>
      
      <b>Reviews and Opinions</b>
      <li><b>Strutz J</b>, Martin J, Greene J, Broadbelt L, Tyo K (2019)
      Metabolic kinetic modeling provides insight into complex biological questions, but hurdles remain.
      <i>Current Opinion in Biotechnology</i>, 59:24-30.</li>
      
      <b>Book Chapters and Technical Reports (Non-peer Reviewed)</b>
      <li>Jeffryes J, <b>Strutz J</b>, Henry C, Tyo KEJ (2019)
      Metabolic in silico network expansions to predict and exploit enzyme promiscuity.
      <i>Methods in Molecular Biology</i>, 1927:11-21.</li>

      <li>Rosenfeld, D, Liew, F, Kopke, M, Gao, A, Harris, A, Mueller, A, Nagaraju, S, Smith, A, Tran, L, O'Brien, J, Tyo, K, Martin, J, <b>Strutz, J</b>, Metz, J. The Dow Chemical Company (2019)
      Bio-syngas to Fatty Alcohols (C6-14) as a Pathway to Fuels.</li>
    </div>

    <!-- Presentations, Career Panels, & Talks -->
    <h1 class="subheading">Presentations, Panels, & Talks</h1>

    <div style="font-size: 16px;">
      <b>November 28, 2023 - Northwestern University Data Science Nights Career Panel - Virtual</b>
      <li>Participated in a career panel for data science students considering a career in industry</li>

      <b>October 27, 2023 - Northwestern University Center for Synthetic Biology Career Panel - Chicago, IL</b>
      <li>Participated in a career panel for synthetic biology graduate students considering a career in industry</li>

      <b>February 24, 2022 - PhD Student Seminar - Evanston, IL</b>
      <li>“Exploring the Uncharted Biochemical Universe: Predicting Metabolites across Scales”</li>
      
      <b>September 10, 2021 - Central US SynBio Workshop - Evanston, IL</b>
      <li>“MINE 2.0: Enhanced biochemical coverage for peak identification in untargeted metabolomics” (poster)</li>

      <b>October 7, 2020 - ARCS Fun at Five - Evanston, IL</b>
      <li>“Using DNA as a medium for recording biological signals”</li>

      <b>September 10, 2020 - Central US SynBio Workshop - Online</b>
      <li>“Kinetic Monte Carlo Modeling of ssDNA Extension via Terminal deoxynucleotidyl Transferase” (poster)</li>

      <b>May 8, 2020 - Virtual Podium 2020 - Online</b>
      <li>“MINEs: Open Access Databases of Computationally Predicted Enzyme Products for Untargeted Metabolomics”</li>

      <b>September 12, 2019 - Central US SynBio Workshop - Madison, WI</b>
      <li>“Prediction of Liquid Chromatography Retention Times for Metabolite Identification” (poster)</li>

      <b>May 21, 2019 - Mini-Symposium on Microbial Lignin Valorization - NREL - Golden, CO</b>
      <li>“Discovery of Novel Metabolites and Gene Functions in Aromatic-Degrading Organisms (Proposed Work)”</li>

      <b>October 31, 2018 - AIChE Annual Meeting - Pittsburgh, PA</b>
      <li>“Addition of Thermodynamic Constraints to a Scalable Kinetic Modeling Framework”</li>

      <b>March 4, 2018 - ARCS Illinois Chapter Meet & Greet - Evanston, IL</b>
      <li>“Improving the Environmental Sustainability of Biorefineries: Engineering a Bacterium to Eat Plants”</li>

      <b>June 26, 2017 - Mini-Symposium on Microbial Lignin Valorization - NREL - Golden, CO</b>
      <li>“Modeling Catabolism of Lignin-Derived Aromatics in Acinetobacter baylyi ADP1”</li>
    </div>

  </div>

</body>

</html>